sbt

```diff
@@ -10,7 +10,7 @@ declare -r sbt_release_version="0.13.17"
 declare -r sbt_unreleased_version="0.13.17"
 
 declare -r latest_213="2.13.0-M5"
-declare -r latest_212="2.12.7"
+declare -r latest_212="2.12.10"
```

Comment:
Unrelated, do we still need this wrapper? It was needed back in the days when sbt wasn't available in CI env?

Reply:
I don’t think so. We used to use it so people didn’t need sbt installed.